recipes/mold/1.4.x/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.15)
+project(mold CXX)
+
+add_executable(mold src/mold.cpp src/main.cpp)
```

Comment:
Here you're bypassing the whole CMakeLists from the mold project. Here you should just add_subdirectory to call that CMakeLists. This file should be just a wrapper around the mold's CMake project.

Reply:
I still think this is wrong. If you're going yo bypass the project's cmake files, then why differentiate between 1.4+ and 1.3 with different recipes?
We should use the build system the project provides, when possible. Bypassing the project's cmakefiles can have unexpected consequences. We could be missing some important define or compile option. And even if we're not doing it now, we could on future releases.

Reply:
original CmakeLists.txt from mold is used currently. The file above does not exist in this MR anymore :)

Reply:
I'm a bad reviewer, sorry!

Reply:
No worries, sometimes it's confusing